grid/social-media/publish-post/profile.supr

```diff
@@ -107,3 +120,47 @@ model MediaItem {
   altText string
 }
 
+// https://developers.facebook.com/docs/instagram-api/reference/error-codes/
+// https://developers.facebook.com/docs/graph-api/guides/error-handling/
+// https://developers.facebook.com/docs/graph-api/overview/rate-limiting#error-codes
+
+model ErrorCode enum {
+  """
+  Bad request
+  The request was invalid or cannot be otherwise served.
+  """
+  BadRequest
+  """
+  Not found
+  The profileId, container or other input item does not exist.
+  """
+  NotFound
+  """
+  Rate limit reached
+  Returned when a request cannot be served due to the user or app reached max requests quota.
+  """
+  RateLimitReached
+  """
+  Unauthenticated
+  There was a problem authenticating your request. Check that you are passing valid access token.
+  """
+  Unauthenticated
+  """
+  Unauthorized
+  The request is understood, but it has been refused or access is not allowed.
+  """
+  Unauthorized
+
+  Timeout
+  ServerError
+  Spam
+  RestrictedProfile
+  PayloadTooLarge
+  UnsupportedFormat
```

Comment:
I wonder if we need both `UnsupportedFormat` and `BadRequest` error codes. Is not this duplicity.

Reply:
Good point, this error relates to the media format (e.g. I am uploading a picture or video with some unsupported encoding).

I will rename this to `UnsupportedAttachmentFormat`